apis/Google.Cloud.Firestore/Google.Cloud.Firestore.Tests/FirestoreDbBuilderTest.cs

```diff
@@ -45,6 +45,27 @@ public void MaximalConfiguration()
             Assert.IsType<CustomConverter<Guid>>(db.SerializationContext.GetConverter(typeof(Guid)));
         }
 
+        [Fact]
+        public void NamedDatabase()

```

Comment:
This is a unit test, but it's using the application default credentials. It's probably simplest to set the CallInvoker to a FakeCallInvoker in the FirestoreDbBuilder below.